Reply/app/src/main/java/com/materialstudies/reply/ui/email/EmailFragment.kt

```diff
@@ -93,10 +94,9 @@ class EmailFragment : Fragment() {
             attachmentAdapter.submitList(email.attachments)
         }
 
-        binding.emailStarImageView.setOnClickListener {
-            EmailStore.update(binding.email!!.id) { isStarred = !isStarred }
-            email.notifyChange()
-        }
+        // Set up the color on the starButton
+        val surfaceColor1 = SurfaceColors.SURFACE_1.getColor(requireContext())
+        binding.emailStarButton.setBackgroundColor(surfaceColor1)
```

Comment:
What do you think about moving this into a BindingAdapter method like `backgroundSurfaceElevation` that accepts a float dimen?

Reply:
I'll try this out. Looks like we have to get the elevation and get the color via. the elevation at that point.

Reply:
Seems like in some situation this does not work well with the ElevationOverlay, will update PR maybe @dsn5ft or you can help find a solution.

Reply:
Figured it out,